pool/pool.py

```diff
@@ -372,6 +382,10 @@ async def create_payment_loop(self):
                 pool_coin_amount = int(total_amount_claimed * self.pool_fee)
                 amount_to_distribute = total_amount_claimed - pool_coin_amount
 
+                if total_amount_claimed < 1.75 * (10 ** 12):
```

Comment:
```suggestion
                if total_amount_claimed < 1.75 * (10 ** 12):
```
Would probably be better to use a unit constant here from `chia-blockchain` instead of `10 ** 12`